kubernetes/cray-service/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v2
 name: cray-service
-version: 7.0.0
+version: 7.1.0
```

Comment:
This change isn't backwards compatible so according to semver https://semver.org/ :

MAJOR version when you make incompatible API changes,

Reply:
I've bumped the MAJOR version.